bundles/mailjet-mail-connector/src/FondOfOryx/Zed/MailjetMailConnector/Business/MailjetMailConnectorBusinessFactory.php

```diff
@@ -29,7 +35,7 @@ public function createMailjetMailer(): MailProviderPluginInterface
      */
     protected function createMailjetClient(): Client
     {
-        return new MailjetClient(
+        $mailjetClient = new MailjetClient(
```

Comment:
you can save the var when returning (new Mail...)->addRequestOption...

Reply:
I don't think so, since `addRequestOption` only returns void and not the instance or am I wrong?

Reply:
oh, not fluid? then ignore my comment :D, sry